.pre-commit-config.yaml

```diff
@@ -29,16 +29,21 @@ repos:
         args: ["--remove"]
       - id: mixed-line-ending
         args: ["--fix=lf"]
+  - repo: https://github.com/jumanjihouse/pre-commit-hooks
+    rev: 2.1.4
+    hooks:
+      - id: markdownlint
+        args: ["-r", "~MD002,~MD013,~MD029,~MD033,~MD034"]
   - repo: https://github.com/myint/docformatter
     rev: v1.3.1
     hooks:
       - id: docformatter
         args: ["--in-place", "--wrap-descriptions", "79"]
-  - repo: local
-    hooks:
-      - id: clang-format
-        name: clang-format
-        description: Format files with ClangFormat
-        entry: clang-format -style=google -i
-        language: system
-        files: \.(c|cc|cxx|cpp|cu|h|hpp|hxx|cuh|proto)$
+  # - repo: local
```

Comment:
Will these commented codes be used in the future? If not, we may clean them.

Reply:
It can be used for local formatting. I comment these lines because of the incompatibility with github actions (requires installing clang-format locally).